content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: An Alternative Istiod Deployment Model
+subtitle: Clear Separation between Mesh Operator and Mesh Admin
```

Comment:
something like "Provide a clear separation of concerns between mesh admins and cluster admins"?

Reply:
the reason I had mesh in the name is because cluster admin may not operate meshes...